hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -45,7 +45,7 @@
  * This class can be serialized and de-serialized and on de-serialization the FileSystem is re-initialized.
  */
 public class HoodieArchivedTimeline extends HoodieDefaultTimeline {
-    private static final String HOODIE_COMMIT_ARCHIVE_LOG_FILE = "commits.archived";
+    private static final String HOODIE_COMMIT_ARCHIVE_LOG_FILE = "commits";
```

Comment:
agree. thats a bad name for a file.. suggested some changes for HoodieLogFile however